configure.ac

```diff
@@ -146,16 +141,16 @@ CHECK_LIBXML2
 
 
 #
-# Check for libpcre
+# Check for libpcre only if explicitly requested
 #
-CHECK_PCRE
-
-
-#
-# Check for pcre2
-#
-PROG_PCRE2
-AM_CONDITIONAL([PCRE2_CFLAGS], [test "PCRE2_CFLAGS" != ""])
+if test "x${with_pcre}" != "x" && test "x${with_pcre}" != "xno"; then
```

Comment:
While this test is correct, it omits the possibility of `with_pcre2` having been set to `no`. It might not make much sense, but if the user sets both `with_pcre` and `with_pcre2` to `no` then I would expect that to be honoured. As it is now, PCRE2 would be used, regardless.